worker/include/RTC/RTCP/CompoundPacket.h

```diff
@@ -12,21 +12,17 @@ namespace RTC { namespace RTCP
 	class CompoundPacket
 	{
 	public:
-		~CompoundPacket();
 
-		uint8_t* GetRaw();
 		size_t GetSize();
```

Comment:
It's just that, for example, `SdesItem` has a `GetSize` and `GetLength`, and I think we use both `GetSize` and `GetLength` for the same purpose in some places.